docs/design/coreclr/jit/first-class-structs.md

```diff
@@ -213,16 +225,21 @@ This would be done in multiple phases:
       for a general idea of the kinds of VM changes that may be required.
   * Defer retyping of struct return types (`Compiler::impFixupStructReturnType()` and
     `Compiler::impFixupCallStructReturn()`)
-    * This is probably the "right" way to fix [#13355](https://github.com/dotnet/runtime/issues/13355).
+    * This has largely (fully?) been addressed with [#37745 Disable JitDoOldStructRetyping by default](https://github.com/dotnet/runtime/pull/37745).
   * Next, eliminate the "pessimizations".
     * For cases where `GT_LCL_FLD` is currently used to "retype" the struct, change it to use *either*
       `GT_LCL_FLD`, if it is already address-taken, or to use a `GT_BITCAST` otherwise.
```

Comment:
I would add https://github.com/dotnet/runtime/issues/11413 to this list.

Reply:
Done, thanks